src/action.ts

```diff
@@ -26,21 +26,27 @@ async function downloadUpx(): Promise<string> {
     throw "unsupported OS";
 }
 
-async function runUpx(file: string, upx_path: string) {
-    child_process.execSync(`strip ${file}`);
-    child_process.execSync(`${upx_path} ${file}`);
-}
-
 export async function run() {
     try {
         const file = core.getInput('file', { required: true });
+        const args = core.getInput('args');
+        const strip = core.getInput('strip') || 'true';
+        const strip_args = core.getInput('strip_args');
 
         if (!fs.existsSync(file)) {
-            core.setFailed(`File ${file} wasn't found.`);
+            core.setFailed(`⛔ File ${file} wasn't found.`);
         }
 
+        if (/true/i.test(strip)) {
```

Comment:
I've never seen this syntax. What is this?

Reply:
It's the safest way to check if a string is a true boolean